Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -129,5 +129,17 @@ private void UpdateStabilizationPlane()
                 HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
             }
         }
+
+        /// <summary>
+        /// Sends message to the object currently focused on.
+        /// </summary>
+        /// <param name="message">Message to send</param>
+        public void SendMessageToFocusedObject(string message)
```

Comment:
GazeManager already contains logic for detecting focusedObject, why not use data which is readily available? Besides, this is an elegant way to avoid leaking the GazeManager abstraction by not exposing FocusedObject outside of GazeManager.